### DISCUSSION TITLE
可以支持树莓派4不

### DISCUSSION THREAD

**Discussion by coolhuasen** [kkkgo/PaoPaoGateWay#57] [问和答]
可以支持树莓派4不？家里的软路由用的是树莓派4B来搭建的。里面有Docker，今天拉镜像，竟然说不支持arm的？有哪个朋友使用过树莓派来搭建不？

**Comment by kkkgo** [comment]
这个docker是用来生成iso镜像用的并不是运行网关，iso镜像是在x86上运行的，所以支持arm没啥意义。

**Reply by kkkgo** [reply]
https://github.com/kkkgo/PaoPaoGateWay/discussions/43#discussioncomment-7703022
